stickytimelineview/src/main/res/values/attrs.xml

```diff
@@ -14,5 +14,9 @@
         <attr name="sectionSubTitleTextSize" format="dimension" />
         <attr name="isSticky" format="boolean" />
         <attr name="customDotDrawable" format="reference" />
+        <attr name="mode" format="enum">
```

Comment:
In my opinion, it would be better if the name is more than `mode`. In general, in case of LinearLayout or RecyclerView, the name is used as `orientation`, so it is better to use a name with` orientation`. (ex: timeLineOrientation)
What do you think?